### PR TITLE
Node-5 support

### PR DESCRIPTION
**Pull request by doug-martin**
- [x] @dustinsmith1024

### CODE REVIEW

File: test/adapters/postgres.test.js [hunk 1]
```diff
@@ -38,46 +37,43 @@ if (process.env.PATIO_DB === "pg") {
                 this.sqls.push(sql.trim());
                 return origExecute.apply(this, arguments);
             };
-            return comb.serial([
```
Comment (dustinsmith1024) [inline test/adapters/postgres.test.js]: can you explain what the issue was?

File: test/associations/manyToMany/camelize.test.js [hunk 2]
```diff
@@ -8,37 +10,22 @@ var it = require('it'),
...
         Company = patio.addModel("company", {
             "static": {
-
                 identifierOutputMethod: "camelize",
-
-                identifierInputMethod: "underscore",
-
-                init: function () {
-                    this._super(arguments);
-                    this.manyToMany("employees");
```
Comment (dustinsmith1024) [inline test/associations/manyToMany/camelize.test.js]: why this change? should we no longer use the init's?

File: lib/sql.js [hunk 3]
```diff
@@ -48,8 +48,7 @@ var virtualRow = function (name) {
     }, function () {
         return SQLFunction.fromArgs(arguments);
     });
-     Object.setPrototypeOf(ident, null);
-     Object.setPrototypeOf(ret, ident);
```
Comment (dustinsmith1024) [inline lib/sql.js]: looks like this was still the main change and the rest was just issues with the tests?

File: test/adapters/postgres.test.js [hunk 4]
```diff
@@ -1,13 +1,11 @@
+"use strict";
+
 var it = require('it'),
     assert = require('assert'),
     patio = require("../../lib"),
     sql = patio.SQL,
-    comb = require("comb-proxy"),
-    config = require("../test.config.js"),
```
Comment (dustinsmith1024) [inline test/adapters/postgres.test.js]: how do i run the tests now?  looks like patio_db needs setting somehow in the ENV.

Comment (vikeen) [inline test/adapters/postgres.test.js]: Here are my environment vars from webstorm

```NODE_PATH=/Users/johnrake/Development/patio/lib;PATIO_DB=mysql```

Comment (vikeen) [inline test/adapters/postgres.test.js]: I think all you need is the PATIO_DB.

File: test/adapters/postgres.test.js (hunk 1 above)
Comment (vikeen) [inline test/adapters/postgres.test.js]: I believe we deprecated comb.serial because using promises returned from patio functions was easier to understand.

File: test/adapters/postgres.test.js [hunk 5]
```diff
@@ -87,7 +83,7 @@ if (process.env.PATIO_DB === "pg") {
         });
 
         it.should("correctly parse the schema", function () {
-            return comb.when(PG_DB.schema("test3"), PG_DB.schema("test4")).chain(function (schemas) {
+            return comb.when([PG_DB.schema("test3"), PG_DB.schema("test4")]).chain(function (schemas) {
```
Comment (vikeen) [inline test/adapters/postgres.test.js]: comb.when will eventually be replaced from a promise implementation such as - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all
